implementations/elixir/ockam/ockam_hub/test/hub/service/alias_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Test.Hub.Service.AliasTest do
+  use ExUnit.Case
+
+  alias Ockam.Hub.Service.Alias, as: AliasService
+  alias Ockam.Hub.Service.Echo, as: EchoService
+  alias Ockam.Router
+  alias Test.Utils
+
+  test "alias test" do
+    {:ok, _worker, worker_address} =
+      Test.Hub.Service.AliasTestWorker.start_link(address: "bdf373h28asdf")
+
+    {:ok, _alias, _alias_address} = AliasService.start_link(address: "cdf373h48asdf")
+    {:ok, _echo, _echo_address} = EchoService.start_link(address: "ddf373h47asdf")
+
+    msg = %{onward_route: [worker_address], return_route: [], payload: Utils.pid_to_string()}
+    Router.route(msg)
+
+    assert_receive(:ok, 5_000)
+  end
+end
+
+defmodule Test.Hub.Service.AliasTestWorker do
+  @moduledoc false
+
+  use Ockam.Worker
+
+  alias Ockam.Message
+  alias Ockam.Router
+  alias Test.Utils
+
+  @echo_address "ddf373h47asdf"
+  @alias_address "cdf373h48asdf"
```

Comment:
We can also use meaningful names in tests, like "alias_address" or "alias_test_service_address"
Also is `@echo_address` still needed? It doesn't seem to be doing anything.